components/eam/bld/configure

```diff
@@ -1239,14 +1239,9 @@ if (defined $opts{'use_MMF'}) {
     if (defined $opts{'crm_adv'}) { $cfg_ref->set('crm_adv', $opts{'crm_adv'}); }
     $cfg_ref->set('MMF_microphysics_scheme', $opts{'MMF_microphysics_scheme'});
     $cfg_ref->set('crm', $opts{'crm'});
-    # Check that model is not using threading for the MMF
```

Comment:
Any CRM code will work with threading now because a PR from last year (I think) removed the entire crm_physics_tend() call from the threaded loop. In the current code, all physics columns from all chunks on an MPI task are combined together, so threading has absolutely no effect on the CRM.